swift/src/Ice/ObjectAdapterI.swift

```diff
@@ -3,14 +3,30 @@
 import IceImpl
 
 class ObjectAdapterI: LocalObject<ICEObjectAdapter>, ObjectAdapter, ICEDispatchAdapter, Hashable {
-    private let communicator: Communicator
     let servantManager: ServantManager
 
+    lazy var dispatchPipeline: Dispatcher = {
+        dispatchPipelineInitialized = true
+
+        // Since this computation is idempotent, we don't need to make it thread-safe. And we don't worry about the
```

Comment:
This is bogus. The middleware in the dispatch pipeline can maintain state and we don't want to ever create & return two distinct pipelines.

Reply:
Fixed.